src/containers/profile/ProfileSections/ManageBlogs/BlogBox.jsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { Link, Redirect } from "react-router-dom";
+import { connect } from "react-redux";
+import { firestoreConnect } from "react-redux-firebase";
+import { compose } from "redux";
+import { updateProfile } from "../../../../actions/authActions";
+import { Dropdown, Button, Row, Col, ListGroup, Image } from "react-bootstrap";
+import "./BlogBox.css";
+import anonymousImage from "../../../../images/anonymousImage.png";
+
+const ADMIN_ROLE = "ADMIN";
```

Comment:
i believe this is being used elsewhere, you should create a file called `common/role-constants.js` and find any other usages of ADMIN_ROLE etc. for example imagine that we want to change the string for ADMIN_ROLE to "ADMINISTRATOR" instead of "ADMIN". right now you would have to change more than one place in the code to achieve this; it should only take a change in one common place